README.md

```diff
@@ -49,6 +52,40 @@ hyprpm enable dynamic-cursors
 
 Compatibility with other plugins is not guaranteed. It probably should work with most plugins, unless they also change your cursor's behaviour. It will however work with any cursor theme.
 
+### NixOS
+
+Add this flake to your inputs, note that using a mismatched/unsupported Hyprland release will fail to build or load the plugin:
+
+```nix
+inputs = {
+    hyprland.url = "git+https://github.com/hyprwm/Hyprland?submodules=1"; # follows development branch of hyprland
+    hypr-dynamic-cursors = {
+        url = "github:VirtCode/hypr-dynamic-cursors";
+        inputs.hyprland.follows = "hyprland"; # to make sure that the plugin is built for the correct version of hyprland
+    };
+};
+```
+
+Then, in your home-manager config, add the plugin:
+
+```nix
+wayland.windowManager.hyprland = {
+    enable = true;
+    plugins = [ inputs.hypr-dynamic-cursors.packages.${pkgs.system}.hypr-dynamic-cursors ];
+};
+```
+
+or add it like this:
+
+```nix
+wayland.windowManager.hyprland = {
+    enable = true;
+    extraConfig = ''
+        plugin = ${inputs.hypr-dynamic-cursors.packages.${pkgs.system}.hypr-dynamic-cursors}/lib/liblibhypr-dynamic-cursors.so
```

Comment:
is `liblib...` intended here? looks like a typo

Reply:
oh god ye I hastily pasted something there at the end, it should be `libhypr-dynamic-cursors.so`